hoomd/Variant.cc

```diff
@@ -3,12 +3,24 @@
 
 #include "Variant.h"
 
-//* Method to enable unit testing of C++ trigger calls from pytest
+/// Method to enable unit testing of C++ variant calls from pytest
```

Comment:
Big picture, what is the plan for testing the C++ API going forward? Are we getting rid of all the C++ tests currently in HOOMD and relying entirely on Python tests (I think that makes sense, just checking)? If so, do we need more tests that directly verify the behavior of the underlying C++ classes?

Reply:
Yes, the plan is to rely on pytest tests wherever possible.